keyboards/boardsource/technik_s/readme.md

```diff
@@ -0,0 +1,21 @@
+# Technik-S
+
+![Technik-S](https://imgur.com/0QLgu5V)
+
+*A short description of the keyboard/project*
+
+* Keyboard Maintainer: [Boardsource](https://github.com/boardsource)
+* Hardware Supported: V1 pcb
+* Hardware Availability: [boardsource](https://boardsource.xyz/store/5ffb9b01edd0447f8023fdb2)
+
+Make example for this keyboard (after setting up your build environment):
```

Comment:
Could you add an example of how to enter the bootloader to flash?

Reply:
Good call. I added a part about it.